openmc/cell.py

```diff
@@ -219,6 +225,14 @@ def paths(self):
         return self._paths
 
     @property
+    def bounding_box(self):
```

Comment:
Is this an expansion of our current bounding box feature? If so, what does this now allow us to do?